src/main/java/edu/hw1/Task1.java

```diff
@@ -0,0 +1,43 @@
+package edu.hw1;
+
+public final class Task1 {
+    private Task1() {
+    }
+
+    private static final int MINUTES_MAX_LEN = 5;
+    private static final int SECONDS_IN_MINUTE_MAX_COUNT = 59;
+    private static final int SECONDS_PER_MINUTE = 60;
+
+    /**
+     *  Возвращает количество секунд из формата записи minutes:seconds.
+     *  Для того, чтобы избежать возможного переполнения, длина minutes
+     * ограничивается в MINUTES_MAX_LEN символов.
+     *
+     * @param time строка в формате minutes:seconds
+     * @return количество секунд или -1 если time не соответсвует требованиям
+     */
+    public static int minutesToSeconds(String time) {
+        String[] timePartition = time.split(":");
+        if (timePartition.length != 2
```

Comment:
Вынеси условия в несколько static boolean методов с человеко-читаемыми названиями
```suggestion
        if (!isValidInput())
```